CaryTick_miceTwo.R

```diff
@@ -0,0 +1,244 @@
+library(rjags)
```

Comment:
These files have no headers explaining what the different models are.  This example, where there's a `CaryTick_miceOne.R` and a `CaryTick_miceTwo.R` is particularly bad as there's no one other than you who would have any idea what these different models are. And I guarantee that in a few months you won't know what they are either.